decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -74,18 +74,29 @@ def external_icon(path, options = {})
       classes = _icon_classes(options) + ["external-icon"]
 
       if path.split(".").last == "svg"
+        icon_path = application_path(path)
+        return unless icon_path
+
```

Comment:
I see that these changes aren't  in the original PR, but they come from  #9636

The problem is that this particular PR we didn't backport to v0.26 (as in that moment we only supported the last release, as we didn't have the backporter script).

Maybe we should first make a backport for that one so we also have the spec and then merge the release branch with this PR?

Reply:
We need those 2 lines, so we will backport #9636 ...

Reply:
@alecslupu I've just merged it, can you merge/rebase and resolve the conflicts 🙏🏽 ?